containers/daap-delete-data-product/src/var/task/delete_data_product.py

```diff
@@ -34,10 +35,14 @@ def handler(event, context):
     logger.add_data_product(data_product_name)
     logger.info(f"event: {event}")
 
-    try:
-        glue_client.delete_database(Name=data_product_name)
-    except glue_client.exceptions.EntityNotFoundException:
-        logger.info(f"Glue database '{data_product_name}' not found.")
+    major_versions = get_all_major_versions(data_product_name=data_product_name)
```

Comment:
(Non-blocking) might be nice to move this to glue_and_athena_utils so we've got a standard way to create/update/delete glue databases and tables.
https://github.com/ministryofjustice/data-platform/blob/main/containers/daap-python-base/src/var/task/glue_and_athena_utils.py

Reply:
Moved to glue & athena utils